R/print.R

```diff
@@ -0,0 +1,15 @@
+# Simple print methods to suppress printing the list element `input` to the
+# console
+
+#' @export
+print.gs_design <- function(x, ...) {
+  x_names <- names(x)
+  for (i in seq_along(x)) {
+    if (x_names[i] != "input") {
+      print(x[i])
+    }
+  }
```

Comment:
Would this work?

```suggestion
  for (i in names(x)) {
    if (i != "input") {
      print(x[i])
    }
  }
```

or even this?

```suggestion
  x$input <- NULL
  print(x)
```

or if we want to follow the convention that a `print()` method usually (invisibly) returns the object to be printed, we can do

```suggestion
  print(x[names(x) != "input"])
  invisible(x)
```

Reply:
Thanks @yihui! Your last suggestion was my favorite